packages/analytics/analytics-chart/src/components/CsvExportModal.cy.ts

```diff
@@ -49,30 +49,15 @@ describe('<CsvExportModal />', () => {
     cy.getTestId('csv-export-modal').find('.k-table thead th').should(th => {
       const elements = Array.from(th, e => e.innerText)
 
-      expect(elements).eql(['Timestamp', 'UtcOffset', 'StatusCode', 'TotalRequests'])
+      expect(elements.join(',')).eql('Timestamp,UTC Offset,Status Code,Request Count')
```

Comment:
The test was complaining, and was also not a reliable check. It was previously simply checking that the two arrays are arrays of length four, rather than actually testing the contents. So I turned it into a string for a better comparison

Reply:
Maybe I’m out of date on tests but what is `.eql()`?

Should this be `.eq()` or `.toEqual()`

Reply:
https://www.chaijs.com/api/bdd/#method_eql

> It was previously simply checking that the two arrays are arrays of length four, rather than actually testing the contents.

I don't see how that can be... and if that was how it was acting, then it's still just checking the length of the string...

Reply:
This is the error I get if I do it the old way....

<img width="313" alt="image" src="https://github.com/Kong/public-ui-components/assets/6026470/2cf1e3fe-6a40-4f26-85fa-84b3c3b38f68">

Reply:
Even though, if you inspect, the contents of the array are the same 🤷

Reply:
The new way... 
<img width="290" alt="image" src="https://github.com/Kong/public-ui-components/assets/6026470/6c6624df-f5d9-45a6-a408-abcb561009f3">


Seems like a better check to me

Reply:
When I run the test on main, it still seems like it's just checking Array(4) against Array(4)? 

<img width="351" alt="image" src="https://github.com/Kong/public-ui-components/assets/6026470/f16acaa1-d596-45e0-ac75-55a4023d5559">

Reply:
> it's just checking Array(4) against Array(4)

Kind of related, I think you should stringify the arrays to compare them instead of comparing actual arrays; I'm not sure you're testing what you think you are.

```ts
expect(JSON.stringify(array1)).toEqual(JSON.stringify(array2))
```

Reply:
Sorry should have clarified -- at least if you stringify and it doesn't match, it will output the strings in the logs instead of `[ Array(4) ]`

Reply:
Which is exactly what creating a string via joining is doing too 😄  

But I agree, using stringify is more elegant